src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -632,6 +632,10 @@ private bool AcquireServerCredentials(ref byte[] thumbPrint, byte[] clientHello)
             X509Certificate localCertificate = null;
             bool cachedCred = false;
 
+            // There are three options for selecting the server certificate. When 
+            // selecting which to use, we prioritize the new ServerCertSelectionDelegate
+            // API. If the new API isn't used we call LocalCertSelectionCallback, and if 
```

Comment:
Add comment to LocalCertSelectionCallback that it is there only for compat with .NET Framework only.

Also we should mention that only LocalCertSelectionCallback and ServerCertificate can be set at once.

Reply:
Nit: I would reword it to:
```c#
// There are three options for selecting the server certificate. When 
// selecting which to use, we prioritize the new ServerCertSelectionDelegate
// API. If the new API isn't used we call LocalCertSelectionCallback (for compat 
// with .NET Framework), and if neither is set we fall back to using ServerCertificate.
```